src/blackboxTest/bash/TestMultiThreading.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+TESTPATH=$1
+EXIT_CODE=0
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+WHITE='\033[0m'
+
+
+runThreads() {
+  ./requestWithDelay.sh ${TESTPATH} > /dev/null 2>/dev/null &
```

Comment:
Right, should probably just pass those in as parameters.

I did the redirect because it didn't seem necessary to have all the text form the telnet commands and then the returned response print to the screen. The person running the test doesn't need to see that.